docs/rules/no-did-update-set-state.md

```diff
@@ -51,7 +51,7 @@ var Hello = React.createClass({
 
 ```js
 ...
-"no-did-update-set-state": [<enabled>, <mode>]
+"no-did-update-set-state": [<enabled>, <modeInFunctions>, <modeViaMethods>]
```

Comment:
Both could be supported with eslint schemas as semver-minor - ie, a "mode" string, and then otherwise, an options object.